sample-code/src/server.ts

```diff
@@ -158,6 +159,14 @@ app.get('/ai-api/models', async (req, res) => {
   }
 });
 
+app.get('/ai-api/deployment-url', async (req, res) => {
+  try {
+    res.send(await getDeploymentUrl('gpt-4o', 'foundation-models', 'default'));
```

Comment:
[q] I also din't get it, since we're exporting the function now, why not directly call it from here? Why duplicate it?

Reply:
Now I see the confusion. True, I now directly use the resolve function.